iOS_SDK/OneSignalSDK/Source/OneSignal.m

```diff
@@ -161,13 +160,12 @@ @implementation OneSignal
 
 static BOOL promptBeforeOpeningPushURLs = false;
 
-
-// Indicates if initialization of the SDK has been delayed until the user gives privacy consent
-static BOOL delayedInitializationForPrivacyConsent = false;
-
 // If initialization is delayed, this object holds params such as the app ID so that the init()
 // method can be called the moment the user provides privacy consent.
-DelayedInitializationParameters *delayedInitParameters;
+DelayedInitializationParameters *_delayedInitParameters;
```

Comment:
Lets keep this consistent with Android SDK class `DelayedConsentInitializationParameters`